### ISSUE TITLE
Sunset Qt Multimedia usage

### ISSUE THREAD

**Issue by KitsuneRal** [quotient-im/libQuotient#483]
Although Qt 6.2 finally has got Qt Multimedia, libQuotient cannot (never could) make proper use of it. To hold the
dependency just for the sake of extracting metadata from an audio/video file in one particular place looks like a waste; and without a GUI it is rather challenging to provide any added value on top of that. Moreover, the `RoomMessageEvent` constructor accepting a `QFileInfo` (the only part of Quotient actually using Qt Multimedia, and even that uses literally one method, `QMediaResource::resolution`, for the video image size) is not extensible: it doesn't support adding new kinds of content which will become a shortcoming once extensible events land in Matrix.

Therefore I think of deprecating this constructor if compiled with Qt 5, and entirely dropping it if compiled with Qt 6 (building with Qt 6 is an experimental configuration anyway for now; and its Qt Multimedia API doesn't allow to read even that image size value synchronously anymore). Clients will have to create an appropriate `EventContent` object themselves. It's likely that they already have everything necessary to extract content metadata if they provide any sort of audio/video/image preview; and even if not, it boils down to making a round over an event loop to read the metadata from the media file.

This would also lower the Qt 6 requirement to from 6.2 to 6.0, although versions before 6.2 are immature and aren't expected to be used much anyway.

**Comment by CarlSchwan** [comment]
Make sense also QMediaResource::resolution didn't really work well for in my experience

**Comment by KitsuneRal** [comment]
Yes, that too.

**Comment by KitsuneRal** [comment]
9a5fa62 broke the build; stay tuned.

**Comment by KitsuneRal** [comment]
...and, while fixing the build breakage, I recollected the reason for the current code. I think Qt Multimedia is here to stay; Quotient just has to make better use of it.

**Comment by redstrate** [comment]
Do we still depend on QtMultimedia? I can't find any usages in the code, only a vestigial mention of it in our `.pri`.

**Comment by KitsuneRal** [comment]
I don't think we do at the moment; but the update in the issue text is still relevant.
